tests/files/_cache/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Why?

Reply:
Putting the `.gitignore` file here was initially made to force creation of directory on git checkout.
Ignoring `test/files/*` can be added to root `.gitignore` file as I added creation of the directory in https://github.com/glpi-project/glpi/pull/5402/files#diff-10eafd8bf5fbdde867b86fe38d30b9bbR513 .

Reply:
Currently, to avoid emtpy directories that are not handled code side, usage seems to rely on a `remove.txt` file.
Event if the code checks for the directory presence, it's certainly not a problem if the directory do exists from checkout.